apps/query-eval/queryeval/driver.py

```diff
@@ -279,11 +281,16 @@ def do_eval(self, _query: QueryEvalQuery, result: QueryEvalResult) -> QueryEvalR
                     console.print(f"Expected node: {diff.node_a!r}")
                     console.print(f"Actual node: [red]{diff.node_b!r}")
                     console.print()
+                metrics.plan_similarity = (len(_query.expected_plan.nodes) - len(plan_diff)) / len(
```

Comment:
Usually I think it's best to capture raw counts rather than aggregate metrics like a ratio, since the ratio can be hard to interpret in some cases (i.e., if a query plan only has one node). Could you capture the number of diffs instead (maybe of of each type)?

Reply:
Let me add both yeah